docs/military/townsville.md

```diff
@@ -4,7 +4,86 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
+| Name               | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| Townsville Approach    | Townsville Approach   | 126.800         | TL_APP                                   |
+| Townsville ADC    | Townsville Tower  | 118.300         | TL_TWR        |
+| Townsville SMC    | Townsville Ground  | 121.800         | TL_GND        |
+| Townsville ACD    | Townsville Delivery  | 128.100         | TL_DEL       |
+| Townsville ATIS    |   | 133.500         | YBTL_ATIS       |
+
+## Airspace
+### TCU
+TL TCU owns the Class C and G airspace within 36 DME TL from `SFC` - `FL180`
+
+<figure markdown>
+![TL TCU/ADC Airspace Structure](img/tltcu.png){ width="2000" }
+  <figcaption>TL ADC/TCU Airspace Structure</figcaption>
+</figure>
+
+### ADC
+TL ADC owns the Class C arispace in the TL CTR (Extends to 7NM of the thresholds of 01 and 19.) from `SFC` - `A015`. This airsapce is designed to facilitate Helicopter Traffic and Ciruit Traffic.
+
+### Surveillance
+TL ADC is permitted to use Surveillance standards for separation. Surveillance coverage can be expected to be available at all levels in the TL ADC arispace. 
+For simulations purposes,  visual separation is assumed to exist belwo the cloud base, and whtin 7NM of the areodrome. Visual separation can still be used to separate from aircraft on an instrument approach, below the cloud base.
+
+## Aerodrome
+### Runway Operations
+Runways 01 and 19 are the primary runways at Townsville. 
+Runway 07 is often used in conjunction with 01 for VFR arrivals and Helicopter Traffic. 
+
+### NON-RNAV Departures
+For non-RNAV approved IFR aircaft with a wake turbulence category of medium or greater, issue a RADAR SID.
+For non-RNAV approved IFR aircraft with a wake turbulence category of light, issue a visual departure or Radar SID
+
+## Charts
 !!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
+## Coordination
+### Enroute
+#### Departures
+Voiceless coordination is in place from TL TCU to KEN(TBP) for aircraft:  
+Planned at or above `F180`: `Assigned F180`  
+Planned below `F180`: `Assigned the RFL`
```

Comment:
Any chance we could simplify this like you did on the KEN page? 

'The Standard assignable level from TL TCU to TBP is the lower of `FL180` or the `RFL`. All other aircraft must be voice coordinated.'